### DISCUSSION TITLE
Update a custom file whenever a dockerfile is updated?

### DISCUSSION THREAD

**Discussion by nsalfos** [renovatebot/renovate#22732] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I have 1 specific Dockerfile: `foo/bar/action-runner/Dockerfile` which has private images inside which are getting picked up for update just fine
Thing is it shares directory with other 2 files (`VERSION` & `CHANGELOG.md` ) which **MUST** be updated anytime a change in the `Dockerfile` is made. (there's a github action enforcing that) 

is there a way to not only updated the VERSION file (which only has 1 string, i.e.: `0.2.3` ==> `0.2.4` ) but also maybe add the  PR title or something to the changelog.md file? 

### What you tried so far.

I've just been reading documentation on `regexManagers` but cannot make sense of how to use properly for the versioning and i don't think i've seen any clues that injecting text in a random file is possible?   

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment, marked as answer]
not possible to add something to the changelog file beside a simple version replacement. You can use self-hosted renovate with post upgrade tasks to do that.

**Comment by rarkins** [comment]
https://docs.renovatebot.com/configuration-options/#postupgradetasks

But you will likely need to self-host for this, e.g. using github action. The hosted app does not allow arbitrary commands as it would be a security risk

**Comment by nsalfos** [comment]
Thanks a mil for the lightning fast response! 
in that case we'll evaluate the possibility to jump into self-hosted in the future and handle the changelog manually for now.
